### PR TITLE
Remove obsolete uses of Strings::escapeTags in mod/item

### LINKED ISSUE #9192: Impossible to insert a '>' into the title of a post
### Expected behavior
When I put something like "... > ..." into the title it should stay that way after submitting the post.

### Actual behavior
When I put something like "... > ..." into the title it is changed to "... ] ..." after submitting the post.

### Steps to reproduce the problem
Enter something containing a `>` into the title of a new post. Look at the preview or submit it and look at the post.

### Friendica version you encountered the problem
2020.09-rc and older

### Friendica source (git, zip)
git

### PHP version
7.4

### SQL version
10.3 MariaDB

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #9192

This is the beginning of a long road towards removing `Strings::escapeTags` that is unfortunately used in hundreds of places throughout the code. The migration problem is that since we've had an overzealous tag escape mechanism on input, we haven't had to make sure the outputs are properly escaped. So each use removal must be paired with a complete understanding where the input will be displayed and whether it is correctly escaped or not.

This was the case for the item `location` attribute that's displayed with the Smarty `nofilter` modifier. So I had to add an `Strings::escapeHtml()` call in this case.

### CODE REVIEW

**Comment by annando** [comment]
What do we do with this PR? Having changed 16 files is making it hard to review, so short before the release.

**Comment by MrPetovan** [comment]
Simply go commit by commit. I lightly resent the fact that you're invoking the short time before the release when you didn't mind when it came to the new AP relay function that is a new and untested feature.

**Comment by annando** [comment]
Since template variables have changed as well, I have to go through all of them, checking if there are abandoned template variables and so on. So it's a step by step code analysis that contain possibly more than the files here in this PR. I hope that I can do it this evening. But because of the upcoming release, I will have to check it more intense than normally.

File: src/Object/Post.php [hunk 1]
```diff
@@ -357,13 +357,12 @@ public function getTemplateData(array $conv_responses, $thread_level = 1)
...
 
-		$body = Item::prepareBody($item, true);
+		$body_html = Item::prepareBody($item, true);
 
 		list($categories, $folders) = DI::contentItem()->determineCategoriesTerms($item);
 
-		$body_e       = $body;
-		$text_e       = strip_tags($body);
 		$name_e       = $profile_name;
+		$text       = strip_tags($body_html);
```
Comment (annando) [inline src/Object/Post.php]: Can you fix the spaces between the two lines?

Comment (annando) [reply src/Object/Post.php]: Hmm, why does the system tells me, that it is outdated?

Comment (annando) [reply src/Object/Post.php]: Ah, okay. The problem are the commits ...

**Pull request merged**